<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortoptionswindow</class>
 <widget class="QDialog" name="sortoptionswindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="applySortButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color: #85ffb2; border: 1px solid black;
     border-radius: 2px;
}



QPushButton:pressed {  
     background-color: #85ffc2; border: 1px solid black;
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:disabled {
    border-color: transparent;
	background-color: #a6ffc7;
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     background-color: #ff8585; border: 1px solid black;
     border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pageName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-weight: 600;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Sorting options</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	font-size: 20px;
border:none;
}</string>
   </property>
   <property name="title">
    <string>Sort using</string>
   </property>
   <widget class="QRadioButton" name="comparisonRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Comparison sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="countingRadio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Counting sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radixRadio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Radix sort</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font-size: 20px;
border: none;
}</string>
   </property>
   <property name="title">
    <string>Sort by</string>
   </property>
   <widget class="QCheckBox" name="idCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="numberCheckBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Train number</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fromCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="toCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="arrTimeCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Arr. time</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rateCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Train rate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="depTimeCheckBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Dep. time</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="typeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>Train type</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="sortTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 19px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sortSequenceLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>811</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 19px;
color: #4a4a4a;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>60</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>165</y>
     <width>961</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>961</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
